### PR TITLE
added gingerbread latte

### PR DESCRIPTION
**Pull request by pratyushakatti13**
Added a new recipe for the holiday season.

### CODE REVIEW

File: src/recipes/gingerbread-latte.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+    "name": "<Gingerbread latte>",
```
Comment (alfg) [inline src/recipes/gingerbread-latte.json:2]: You can remove the `<` and `>`, these are just part of the template.

File: src/recipes/gingerbread-latte.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            "quantity": "",
+            "measure": "",
+            "ingredient": "whipped cream and mini gingerbread men, to serve"
+        }
+    ],
+    "directions": [
+        "In a bowl, mix together the spices, sugar and vanilla extract. Heat 100ml of the milk in a pan with the spice and sugar mix, whisking until the sugar has dissolved. Whisk in the remaining milk in additions until incorporated, then heat through  until steaming.",
+        "Put a shot of espresso in each glass (if using) and top up with the spiced milk. Top with whipped cream, extra spices and a mini gingerbread man. "
+    ],
+    "image": "gingerbread-latte.jpg",
```
Comment (alfg) [inline src/recipes/gingerbread-latte.json:51]: Can you add this image to the PR? Thank you.

**Review by alfg — CHANGES_REQUESTED** [review]

File: src/recipes/gingerbread-latte.json (hunk 2 above)
Comment (pratyushakatti13) [reply src/recipes/gingerbread-latte.json:51]: Yes. Will do that

Comment (pratyushakatti13) [reply src/recipes/gingerbread-latte.json:51]: Hey, I have changed the file and created new pull request regarding the image. Please Review it. Thank you!

Comment (alfg) [reply src/recipes/gingerbread-latte.json:51]: Hi @pratamawijaya, can you add it to this PR?

**Comment by alfg** [comment]
Thanks!

**Pull request merged**